demo/common/overview.c

```diff
@@ -214,8 +214,12 @@ overview(struct nk_context *ctx)
                 nk_button_symbol(ctx, NK_SYMBOL_TRIANGLE_RIGHT_OUTLINE);
 
                 nk_layout_row_static(ctx, 30, 100, 2);
+                nk_style_push_flags(ctx, &ctx->style.button.text_alignment, NK_TEXT_LEFT);
                 nk_button_symbol_label(ctx, NK_SYMBOL_TRIANGLE_LEFT, "prev", NK_TEXT_RIGHT);
+                nk_style_pop_flags(ctx);
```

Comment:
Overall, I think this looks great! My only worry is that when reading it, we set text alignment to left, and then text alignment to right. While I understand why, it does seem confusing.

Would really appreciate more :eyes: from others around here before we merge.

Reply:
Good point... Perhaps with the flags, that could make sense :thinking: